scripts/nasa/cmip6_sea_level/process.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import glob
+import os
+import sys
+
+import numpy as np
+import xarray
+from absl import app
+from absl import flags
+
```

Comment:
please add a top-level comment for the module, especially for future maintainability

Reply:
Done.